sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -77,13 +77,13 @@ export default defineComponent({
     const assessmentDataLoaded = (
       applicationStatus: ApplicationStatus,
       noaApprovalStatus: AssessmentStatus,
-      assessmentId: number,
+      currentAssessmentID: number,
     ) => {
       viewOnly.value = !(
         applicationStatus === ApplicationStatus.Assessment &&
         noaApprovalStatus === AssessmentStatus.required
       );
-      currentAssessmentId.value = assessmentId;
+      currentAssessmentId.value = currentAssessmentID;
```

Comment:
Please adjust the case in the "ID", it should be `currentAssessmentId`.

Reply:
The reason I kept making different names (like the previous currAssessmentId) was to keep the three variables named distinctly. I'd changed this one to be different from the other assessmentId on the page, and different than currentAssessmentId so there wasn't currentAssessmentId.value = currentAssessmentId. I can change it back to assessmentId as that overlap at least is slightly less confusing.